vllm/compilation/decorators.py

```diff
@@ -247,19 +247,42 @@ def __init__(self, *, vllm_config: VllmConfig, prefix: str = "", **kwargs):
             return
 
         compilation_counter.num_models_seen += 1
-        TorchCompileWrapperWithCustomDispatcher.__init__(
-            self, compilation_mode=vllm_config.compilation_config.mode
-        )
+        self.compiled = False
```

Comment:
This is the "have we already done one compile" flag? Can we name it something more descriptive if so? "self.first_compile_done"